helpers.py

```diff
@@ -12,19 +13,30 @@
 EXCLUDE_FROM_BOM = 3
 
 
-def is_version8(version: str) -> bool:
-    """Check if version is 8 or 8 Nightly build."""
-    return any(v in version for v in ("8.0", "8.99"))
+def _is_version_in_range(version: str, min_version: str, max_version: str) -> bool:
+    """Check if version is in range."""
+    ver = LooseVersion(version)
+    return LooseVersion(min_version) <= ver < LooseVersion(max_version)
+
+
+# def is_version9(version: str) -> bool:
```

Comment:
if these aren't being used its ok to remove them, I think is_version6 and is_version7 are suitable templates for a v8 or v9. version.

Reply:
I guess this future preparation won't harm.. If we wanted to distinguish between versions we don't have to search for the specifics numbers.